KPI-team1-frontend/src/App.tsx

```diff
@@ -38,8 +44,7 @@ export default function App() {
     try {
       const { data, error } = await supabase
         .from("circle")
-        .select("circle_name, circle_user!inner(*)")
-        .eq("circle_user.user_id", user.id);
+        .select("circle_name, circle_user!inner(*)");
```

Comment:
Hmmm, selection of default circles depends on this and it needs to include only circles to which the user is assigned to. Could we create a separate state var for this?

Reply:
I reverted to the old code, and this change doesn't affect the other part so everything is ok.